Dockerfile

```diff
@@ -63,6 +61,8 @@ COPY --from=bigtable-emulator-build /go/bin/emulator /app/venv/bin/cbtemulator
 COPY override/gcloud /app/venv/bin/gcloud
 COPY override/timeout.conf /etc/nginx/conf.d/timeout.conf
 COPY override/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+# Hack to get zstandard from PyPI - remove if conda-forge linked lib issue is resolved
```

Comment:
The direct cause of the conda-forge issue is https://github.com/conda-forge/zstandard-feedstock/pull/45, it choose to link the system zstd to solve the license issue mentioned in https://github.com/conda-forge/zstandard-feedstock/issues/43

Reply:
The `multi_decompress_to_buffer` function uses internal pool api, which is only enabled when the package is compiled with bundled zstd.